README.md

```diff
@@ -99,8 +99,10 @@ cirrus validate
 ## Caching
 
 By default, Cirrus CLI stores blob artifacts produced by the [cache instruction](https://cirrus-ci.org/guide/writing-tasks/#cache-instruction)
-in the [user-specific cached data folder](https://golang.org/pkg/os/#UserCacheDir). [Similar to Cirrus Cloud](https://cirrus-ci.org/guide/writing-tasks/#http-cache)
-the CLI can use a caching HTTP server for more efficient sharing of cached artifacts between tasks executed on different physical hosts.
+in the [user-specific cached data folder](https://golang.org/pkg/os/#UserCacheDir).
+Please note, that if you run `cirrus` under `sudo` (for example, in ArchLinux by default), check `/root` directories.
```

Comment:
>if you run `cirrus` under `sudo` (for example, in ArchLinux by default)

Care to elaborate on why is this a default for Arch Linux?

Reply:
I guess, security reasons. From the ArchLinux Wiki:

> If you want to be able to run the `docker` CLI command as a non-root user, add your user to the `docker` [user group](https://wiki.archlinux.org/title/User_group), re-login, and restart `docker.service`.

>
> Warning: Anyone added to the `docker` group is root equivalent because they can use the `docker run --privileged` command to start containers with root privileges. For more information see [[3]](https://github.com/docker/docker/issues/9976) and [[4]](https://docs.docker.com/engine/security/security/).

Reply:
Sorry, but I fail to see the how your quote implies that the `sudo` is default/should be used on Arch Linux. As [we've previously discussed](https://github.com/cirruslabs/cirrus-cli/issues/74#issuecomment-690300414), consider running Cirrus CLI as a separate user and adding that user to the `docker` group instead if you're concerned about the `docker` attack vector.

Also, by default, the `sudo` invocation doesn't seem to actually change the `$HOME`:

```
% echo $HOME
/Users/edi
% sudo sh -c 'echo $HOME'   
/Users/edi
```

Do you have a different behavior on your system?